data/MI/incentives.json

```diff
@@ -64,7 +64,7 @@
     }
   },
   {
-    "id": "MI-25",
+    "id": "MI-22",
```

Comment:
I wasn't treating these IDs as enduring personally. I do briefly remember this coming up, but I've noticed that when adding and deleting rows or when working with partners who do so. It is often required to re-apply IDs, which in some cases it can be pretty tedious not to just lean into the sheet to create a new sequence. I did that here. 

I guess I'm wondering if there are other reasons why you think the IDs should be preserved? I'm currently working towards a solution where we no longer leverage the spreadsheets and hopefully this will be less of an issue although that work is still TBD. 

@oyamauchi - correct me if I'm wrong here about the disposability of the incentive IDs.

Reply:
I haven't been renumbering existing incentives, mainly because they're referred to from elsewhere: relationships files, low income thresholds, and geo groups all refer to incentives by ID. There are some checks in place to try to prevent mismatches in those references, but they're not perfect. Free text fields in spreadsheets, as well as Asana tasks, also refer to incentives by ID.

So, rather than deleting rows, I just mark them as "omit from API" and briefly explain why in whatever notes/comments column there is. And I only add rows at the end, even though that means closely related incentives may get non-consecutive numbers.

Reply:
I see, I guess for the states I've been working in a lot of that hasn't been established yet or I was involved in setting it up so I'd also know where to go and update the data.

Reply:
FWIW here, I did check geo groups and low_income_thresholds to and updated those references.

Reply:
In this [doc](https://docs.google.com/spreadsheets/d/1BJRTPSxvz8IPpbIjAwdlCX78S15hilFPFH8oONiG_zI/edit?usp=sharing), we ended up pulling all of the `omit from API` rows to another sheet. As there was incomplete/problematic data in the sheet that was causing some issues for this script `generate-misc-state-data.ts` to run, even with the flag.

Reply:
So, I think I got this back to the previous state where the IDs are preserved, but I'm going to throw this issue into the drawer of reasons why I don't like leveraging JSON nor the spreadsheet 😂.